app/views/departments/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to("<< Back to Departments", departments_path, :class => 'back-link') %>
+
+<div class ="departments new">
+  <h2>Create New Department</h2>
+  <%= simple_form_for @department do |f| %>
+      <%= f.error_notification %>
+
+      <%= f.input :name %>
+      <%= f.input :position %>
+      <%= f.input :head_name %>
+      <%= f.input :description %>
+      <%= f.input :capacity %>
+
+      <%= f.button :submit %>
+  <% end %>
+</div>
```

Comment:
Here , the red marked error is showing because we need to provide an extra line at the very end of a file. SO just add a single line to avoid this error

Reply:
![screen shot 2018-09-19 at 11 35 22 am](https://user-images.githubusercontent.com/39872561/45732883-1f8a9d80-bc00-11e8-8b53-6169e7509488.png)
Please see the attached screenshot.